site/content/book/v2/didrotation.md

```diff
@@ -0,0 +1,33 @@
+## DID Rotation
+
+[DIDComm Messaging](https://identity.foundation/didcomm-messaging/spec/) relies on the [DIDs](https://www.w3.org/TR/did-core/), and associated [DID Documents](https://www.w3.org/TR/did-core/#dfn-did-documents), of the two peers involved in the communication. Those parties are responsible to secure their keys and perform key rotations if they are compromised, and to make other changes in the DID Documents if needed such as service endpoints. DIDComm has no involvement in that procedure that is particular to each DID method; nevertheless and provided that the DID does not change, the communication can keep flowing normally.
```

Comment:
This is a minor nit, but key rotations can and should happen even if they are not compromised. Adding just a bit to acknowledge that will prevent it from detracting from the larger work you've contributed.